std/meta.d

```diff
@@ -60,6 +60,9 @@ unittest
     static assert(is(Types == AliasSeq!(int, double, char)));
 }
 
+private enum isUDT(T) = is(T == struct) || is(T == class) || is(T == interface);
```

Comment:
does this need to handle `union` as well? `isUDT` look like `isAggregateType` minus `(T == union)`.